ambry-api/src/main/java/com/github/ambry/config/ReplicationConfig.java

```diff
@@ -101,6 +101,25 @@ public class ReplicationConfig {
   @Default("0")
   public final long replicationInterReplicaThreadThrottleSleepDurationMs;
 
+  @Config("replication.enable.acyclic.replication")
+  @Default("true")
+  public final boolean replicationEnableAcyclicReplication;
+  /**
+   * The max number of times a replication group will be considered for iteration in a cycle
+   * in an intra colo replica thread
+   */
+  @Config("replication.intra.max.iterations.per.replication.group.per.cycle")
+  @Default("1")
+  public final int replicationIntraMaxIterationsPerReplicationGroupPerCycle;
+
```

Comment:
Do we plan to handle inter and intra colo replication iterations separately? In what scenarios , these values will be different?

Reply:
+1, let merge the two configs and have one like replication.max.iter.per.cycle

Reply:
I am planning to test both threads separately, so we need both configs currently. We will deprecate these two configs after doing some testing internally.

Reply:
Will merge it to single config as suggested by @snalli after deprecating these 2.

Reply:
One way you can test both intra and inter-colo separately is by setting the the num threads of intra and inter-colo separately. Set one to 0 and other to non-zero. Does this work for your testing ?

Reply:
We will also need to test it in production, we cannot set thread count to 0 there.